.github/workflows/my-ci-pipeline.yml

```diff
@@ -0,0 +1,44 @@
+name: Continuous Integration
+on:
+  push:
+    paths-ignore:
+      - "README.md"
+  pull_request:
+    # Only on pull requests to main
+    branches:
+      - main
+    paths-ignore:
+      - "README.md"
+  schedule:
+    # Run every Sunday at 00:00 UTC
+    - cron: "0 0 * * 0"
```

Comment:
Took me a moment to work out why I couldn't see any evidence this was running, but it's only on the default branch (`main`) so that makes sense. Nice addition 👍